### PR TITLE
fix addTrack/removeTrack

### PR DESCRIPTION
**Pull request by fippo**
the previous code didn't work

### CODE REVIEW

File: rtcstats.js [hunk 1]
```diff
@@ -212,6 +212,29 @@ module.exports = function(wsURL, getStatsInterval, prefixesToWrap) {
         }
       });
 
+      ['addTrack'].forEach(function(method) {
+        if (origPeerConnection.prototype[method]) {
+          var nativeMethod = pc[method];
+          pc[method] = function() {
+            var track = arguments[0];
+            var streams = [].slice.call(arguments, 1);
+            trace(method, id, track.kind + ':' + track.id + ' ' + streams.map(function(s) { return 'stream:' + s.id; }));
```
Comment (ggarber) [inline rtcstats.js]: how is a list serialized to string in JS?   shouldn't you do a .join(' ') and force whatever separator you want?

File: rtcstats.js [hunk 2]
```diff
@@ -212,6 +212,29 @@ module.exports = function(wsURL, getStatsInterval, prefixesToWrap) {
...
+          };
+        }
+      });
+
+      ['removeTrack'].forEach(function(method) {
+        if (origPeerConnection.prototype[method]) {
+          var nativeMethod = pc[method];
+          pc[method] = function() {
+            var track = arguments[0].track;
+            trace(method, id, track ? track.kind + ' ' + track.id : 'null');
```
Comment (ggarber) [inline rtcstats.js]: It is track.kind : track.id above   (different separator).

**Comment by ggarber** [comment]
LGTM

File: rtcstats.js (hunk 1 above)
Comment (fippo) [reply rtcstats.js]: for robustness yes! It is currently coerced into a string but...

**Pull request merged**